javascript/javascript1/week3/lifespan.js

```diff
@@ -0,0 +1,33 @@
+const seriesDurations = [{
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+const totalHours = 365.25 * 80 * 24
+
+function logOutSeriesText(seriesDurations) {
+  let totalTimeSpend = 0;
+  for (let i = 0; i < seriesDurations.length; i++)
+   {
+    let minutes = (seriesDurations[i].days * 24) + seriesDurations[i].hours + (seriesDurations[i].minutes / 60);
```

Comment:
why is this variable called minutes?

Reply:
Actually I thought of converting everything to minutes but changed to hours. Now I changed to let hours.

Reply:
Changed now